src/components/Item.js

```diff
@@ -24,8 +24,28 @@ const Item = ({ item, handleChange }) => {
     setChecked(check);
   }, [item]);
 
+  const addNextPurchaseStyling = item => {
+    let color = '';
+
+    if (item.status === 'inactive') {
+      color = '#747d8c';
+    } else {
+      if (item.nextPurchase <= 7) {
+        color = '#eb2f06';
+      } else if (item.nextPurchase > 7 && item.nextPurchase < 30) {
+        color = '#eccc68';
+      } else if (item.nextPurchase >= 30) {
+        color = '#2ed573';
```

Comment:
I might recommend pulling these hex's out into their own variables that are named after the color themselves. These are referred to as magic numbers and by creating a variable, the code becomes more readable and it will make refactoring easier in the future should you decide on a color theme and need to change these colors across the code base.